cinema/urls.py

```diff
@@ -1,10 +1,45 @@
-from django.urls import path
+from django.urls import path, include
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    CinemaHallViewSet,
+    MovieViewSet
+)
+
+from rest_framework import routers
+
+cinema_hall_list = CinemaHallViewSet.as_view(
+    actions={"get": "list",
+             "post": "create"}
+)
+cinema_hall_detail = CinemaHallViewSet.as_view({
+    "get": "retrieve",
+    "put": "update",
+    "patch": "partial_update",
+    "delete": "destroy",
+})
+
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
 
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("", include(router.urls)),
```

Comment:
No need to include router.urls if you does not use base path
urlpatters = [some urls] + router.urls